settings.php

```diff
@@ -41,4 +41,8 @@
     $settings->add(new admin_setting_configcheckbox('logstore_xapi/backgroundmode',
         get_string('backgroundmode', 'logstore_xapi'),
         get_string('backgroundmode_desc', 'logstore_xapi'), 0));
+
+    $settings->add(new admin_setting_configtext('logstore_xapi/maxbatchsize',
```

Comment:
We should consider that it may be more than just Moodle pushing statements to the LRS. For example, imagine the LRS is not load balanced, if Moodle sends 500 statements just before another service X sends 1 statement, X may have to wait a long time while Moodle hogs the LRS. On the other hand, if Moodle sends 20 statements, X will have far less time to wait.